Cargo.toml

```diff
@@ -117,9 +117,9 @@ sha3 = "0.10"
 # This one needs to match the version used by blockifier
 starknet_api = "=0.10.0"
 syn = "1.0"
+thiserror = "1.0.48"
```

Comment:
This moved and is no longer alphabetically ordered 🤷 Unsure if we care much.

Reply:
That's probably me doing conflict resolution incorrectly. I'll fix this.